src/stream-dispatcher/src/dispatcher/k8_dispatcher.rs

```diff
@@ -171,7 +171,7 @@ where
                         },
                         Err(err) => {
                             error!("WS channel error: {}", err);
```

Comment:
since we have panic, we don't need error?

Reply:
You know, I dunno. It was like that when I got here. Do you want me to remove that? I mean it doesn't hurt at the very least.

Actually, thinking about it more, if we at some point add more tracing modules such as opentelemetry, this would report the error to a remote system before the panic happens. So I think it's worth keeping. 🤷‍♂️

Reply:
Thats good point